File: fixtures/cli-exit-code-projects/config-raises/.github/ghagen_workflows.py
"""Exit-code fixture: a config module that raises at import time.

Discovered by the Python port via ``CONFIG_SEARCH_PATHS``
(``.github/ghagen_workflows.py``). The TypeScript port never sees this file --
its own search paths are disjoint -- so the sibling ``ghagen.workflows.ts``
is the TypeScript half of the same fixture.

Deliberately raises a plain ``RuntimeError``: the contract row this backs
(``config-module-raises``) is about an *arbitrary* exception escaping a
command, not about any error type the CLI knows.
"""

raise RuntimeError("ghagen exit-code fixture: the config module raised")

File: fixtures/cli-exit-code-projects/config-raises/ghagen.workflows.ts
/**
 * Exit-code fixture: a config module that throws at import time.
 *
 * Discovered by the TypeScript port via `CONFIG_SEARCH_PATHS`
 * (`ghagen.workflows.ts`). The Python port never sees this file -- its own
 * search paths are disjoint -- so the sibling `.github/ghagen_workflows.py`
 * is the Python half of the same fixture.
 *
 * Imports nothing on purpose: the fixture is copied into a temp directory
 * where `@ghagen/ghagen` would not resolve, and throwing before any import
 * keeps the row independent of module resolution.
 */

throw new Error("ghagen exit-code fixture: the config module raised");
